CMakeLists.txt

```diff
@@ -490,6 +490,7 @@ if(USE_TCMALLOC)
         CXXFLAGS=${THIRD_PARTY_CXX_FLAGS}
     BUILD_COMMAND make
     BUILD_IN_SOURCE 0
+    BUILD_BYPRODUCTS <INSTALL_DIR>/lib/libtcmalloc_minimal.a
```

Comment:
Yes and no.
Yes `<INSTALL_DIR>` refers to the value of `PREFIX`. No we don't have the liberty to rename it, it's a special string that gets replaced by `ExternalProject` module